charts/landscaper/charts/landscaper/Chart.yaml

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 apiVersion: v2
-name: landscaper
+name: landscaper-controller
 description: A Helm chart for Kubernetes
```

Comment:
While we are at it, could you maybe also add a more "descriptive" description? :-)